Makefile

```diff
@@ -209,14 +209,16 @@ dev_data: git
 	cd $(WPTD_GO_PATH)/util; go get -t ./...
 	go run $(WPTD_GO_PATH)/util/populate_dev_data.go $(FLAGS)
 
-deploy_staging: gcloud webapp_deps package_service var-BRANCH_NAME var-APP_PATH var-PROJECT $(WPTD_PATH)client-secret.json
-	gcloud config set project $(PROJECT)
+gcloud-login: gcloud
 	gcloud auth activate-service-account --key-file $(WPTD_PATH)client-secret.json
+
+deploy_staging: gcloud-login webapp_deps package_service var-BRANCH_NAME var-APP_PATH var-PROJECT $(WPTD_PATH)client-secret.json
```

Comment:
Please move the dependency of `$(WPTD_PATH)client-secret.json` to `gcloud-login`.

Reply:
Done.